com.unity.netcode.gameobjects/Runtime/Core/NetworkObject.cs

```diff
@@ -165,6 +193,26 @@ internal set
         /// </summary>
         public bool AutoObjectParentSync = true;
 
+        public InterestSettings InterestSettingsOverride;
```

Comment:
What exactly is the use case for overriding these on a per NetworkObject level?

Reply:
Well, in the same way that we now have you bind your InterestNode to a NetworkObject instance (vs. the archetype of the NO), we let you customize interest settings per object too.  Quite honestly this is a placeholder / reserving a place more than anything else.

However, in typing this it occurs to me I may want to move the InterestSettings to be under `IInterestObject` to decouple more from NetworkObject